src/foam/dao/Relationship.js

```diff
@@ -74,13 +78,15 @@ foam.CLASS({
     {
       class: 'FObjectArray',
       name: 'sourceProperties',
-      of: 'Property',
```

Comment:
Property changed to Property info for Javaa generation, could this cause issues on the JS side?

`javaType` alone won't suffice as property array.